frontend/src/app/pages/contact/contact.component.scss

```diff
@@ -0,0 +1,44 @@
+@import 'mixins';
+@import '_colors';
+@import 'fonts';
+@import 'button';
+
+:host {
+  @include container;
+  display: block;
+  padding-top: 60px;
+  padding-bottom: 60px;
+  text-align: center;
+}
+
+.map {
+  width: 100%;
+  height: 500px;
+
+  @include for-phone-intermediate {
+    height: 400px;
+  }
+
+  @include for-phone {
+    height: 250px;
+  }
+}
+
+.feedback {
+  margin-top: 50px;
+
+  &__form {
+    display: inline-block;
+    width: 500px;
+    max-width: 100%;
+  }
+}
+
+.button {
```

Comment:
Change class name. It could call naming collision with `.button` in shared/styles/_button.scss